FTBF fpr aarch64-linux

```
# github.com/ProtonMail/proton-bridge/v3/cmd/Desktop-Bridge
/nix/store/8mkiaqipr8d434kbs0ah0my6k4m002j1-go-1.19.7/share/go/pkg/tool/linux_arm64/link: running g++ failed: exit status 1
/nix/store/didj9sv4gfv0dgq2k5hrayr4r5jpkfl9-binutils-2.40/bin/ld: cannot find -lparser: No such file or directory
/nix/store/didj9sv4gfv0dgq2k5hrayr4r5jpkfl9-binutils-2.40/bin/ld: cannot find -lparser_gen: No such file or directory
/nix/store/didj9sv4gfv0dgq2k5hrayr4r5jpkfl9-binutils-2.40/bin/ld: cannot find -lproto: No such file or directory
/nix/store/didj9sv4gfv0dgq2k5hrayr4r5jpkfl9-binutils-2.40/bin/ld: cannot find -lprotobuf: No such file or directory
/nix/store/didj9sv4gfv0dgq2k5hrayr4r5jpkfl9-binutils-2.40/bin/ld: cannot find -lantlr4-runtime: No such file or directory
collect2: error: ld returned 1 exit status
error: builder for '/nix/store/7j8rprgdnrq59a3qf8jahkvmsxhxqw0w-protonmail-bridge-3.0.21.drv' failed with exit code 1;
```

--- a/nixos/doc/manual/release-notes/rl-2305.section.md
+++ b/nixos/doc/manual/release-notes/rl-2305.section.md
@@ -134,7 +134,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The [services.wordpress.sites.&lt;name&gt;.plugins](#opt-services.wordpress.sites._name_.plugins) and [services.wordpress.sites.&lt;name&gt;.themes](#opt-services.wordpress.sites._name_.themes) options have been converted from sets to attribute sets to allow for consumers to specify explicit install paths via attribute name.
 
-- `protonmail-bridge` package has been updated to v3.0 and the CLI executable is now named bridge instead of protonmail-bridge to be more in line with upstream.
+- `protonmail-bridge` package has been updated to major version 3.
 
 - Nebula now runs as a system user and group created for each nebula network, using the `CAP_NET_ADMIN` ambient capability on launch rather than starting as root. Ensure that any files each Nebula instance needs to access are owned by the correct user and group, by default `nebula-${networkName}`.
 
@@ -285,8 +285,6 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The new option `programs.singularity.enableFakeroot`, if set to `true`, provides `--fakeroot` support for `apptainer` and `singularity`.
 
 - The `unifi-poller` package and corresponding NixOS module have been renamed to `unpoller` to match upstream.
-
-- `protonmail-bridge` package has been updated to v3.0 and the CLI executable is now named bridge instead of protonmail-bridge to be more in line with upstream.
 
 - The new option `services.tailscale.useRoutingFeatures` controls various settings for using Tailscale features like exit nodes and subnet routers. If you wish to use your machine as an exit node, you can set this setting to `server`, otherwise if you wish to use an exit node you can set this setting to `client`. The strict RPF warning has been removed as the RPF will be loosened automatically based on the value of this setting.
 

--- a/pkgs/applications/networking/protonmail-bridge/default.nix
+++ b/pkgs/applications/networking/protonmail-bridge/default.nix
@@ -38,7 +38,7 @@ buildGoModule rec {
   ];
 
   postInstall = ''
-    mv $out/bin/Desktop-Bridge $out/bin/bridge # The cli is named like that in the upstream repo
+    mv $out/bin/Desktop-Bridge $out/bin/protonmail-bridge # The cli is named like that in other distro packages
   '';
 
   meta = with lib; {

